Gir.toml

```diff
@@ -11,35 +11,68 @@ single_version_file = true
 
 generate = [
     "Atk.Action",
-    # "Atk.Attribute",
-    # "Atk.AttributeSet",
     "Atk.Component",
+    "Atk.CoordType",
     "Atk.Document",
+    "Atk.EventListener",
+    "Atk.EventListenerInit",
+    "Atk.FocusHandler",
+    "Atk.GObjectAccessible",
+    "Atk.Hyperlink",
+    "Atk.HyperlinkImpl",
+    "Atk.HyperlinkStateFlags",
     "Atk.Hypertext",
     "Atk.Image",
+    # "Atk.Implementor",
+    # "Atk.KeyEventStruct",
     "Atk.Layer",
     "Atk.Misc",
+    # "Atk.NoOpObject",
+    "Atk.NoOpObjectFactory",
+    "Atk.ObjectFactory",
+    "Atk.Plug",
+    "Atk.PropertyChangeHandler",
+    # "Atk.PropertyValues",
     "Atk.Range",
     "Atk.Rectangle",
     "Atk.Registry",
+    "Atk.Relation",
+    "Atk.RelationType",
+    "Atk.RelationSet",
     "Atk.Role",
+    "Atk.Selection",
+    "Atk.Socket",
     "Atk.State",
-    "Atk.Table",
+    "Atk.StateType",
+    "Atk.StateSet",
+    "Atk.StreamableContent",
     "Atk.TableCell",
     "Atk.Text",
```

Comment:
"Atk.Text" this need be removed, as it in separate section

Reply:
After remove and regen `free_ranges` problem "fixed"

Reply:
Forgot to regen before pushing, my bad...